ark/segmentation/marker_quantification.py

```diff
@@ -255,6 +283,10 @@ def generate_cell_table(segmentation_labels, tiff_dir, img_sub_folder,
             necessary for speed and memory considerations
         dtype (str/type):
             data type of base images
+        extraction (str):
+            extraction function used to compute marker counts. default is 'default'
+        **kwargs:
+            arbitrary keyword arguments
```

Comment:
We should validate that extraction belongs to a list of implemented extraction functions